src/components/side-menu/BattleControls.vue

```diff
@@ -44,13 +44,14 @@ onKeydown((ev) => {
   //   return true
   // }
 
-  if (ev.code == 'KeyA') {
+
```

Comment:
Had to change these as well, but the other way, so users can use the keys they see on the keyboard :D